packages/playground/website/src/lib/redux-store.ts

```diff
@@ -61,44 +69,52 @@ const initialState: AppState = {
 			? 'mount-markdown-directory'
 			: null,
 	offline: !navigator.onLine,
+	clients: {},
 	siteListing: {
 		status: { type: 'loading' },
 		sites: [],
 	},
 };
 
-if (query.get('storage') === 'browser') {
-	const siteSlug = query.get('site-slug') || 'wordpress';
-	const opfsRoot = await navigator.storage.getDirectory();
-	const opfsDir = await opfsRoot.getDirectoryHandle(
-		siteSlug === 'wordpress' ? siteSlug : 'site-' + siteSlug,
-		{
-			create: true,
-		}
-	);
-
-	initialState.opfsMountDescriptor = {
-		device: {
-			type: 'opfs',
-			path: await directoryHandleToOpfsPath(opfsDir),
-		},
-		mountpoint: '/wordpress',
-	};
-}
-
 // Create the slice
 const slice = createSlice({
 	name: 'app',
 	initialState,
-	selectors: {
-		getOpfsHandle: (state) => state.opfsMountDescriptor,
-	},
 	reducers: {
-		setPlaygroundClient: (
+		setActiveSite: (state, action: PayloadAction<SiteInfo>) => {
+			state.activeSite = action.payload;
+			state.opfsMountDescriptor = {
+				device: {
+					type: 'opfs',
+					path: '/' + getDirectoryNameForSlug(action.payload.slug),
+				},
+				mountpoint: '/wordpress',
+			};
+		},
+		forgetClientInfo: (state, action: PayloadAction<string>) => {
+			delete state.clients[action.payload];
+		},
+		setClientInfo: (
```

Comment:
It's really cool that this structure acknowledges the fact that different clients can be coming and going depending on UI actions.